server/views/pages/home-new.html

```diff
@@ -41,10 +42,11 @@ <h2 class="govuk-heading-l govuk-hub-white-heading govuk-!-margin-top-5 govuk-!-
         </div>
       </div>
     </div>
-    <div class="govuk-grid-column-one-quarter govuk-!-padding-0">
-      <!-- 300px column, updates small guide links -->
-      Guide links
-    </div>
+    {% if keyInfo.data | length %}
+      <div class="govuk-grid-column-one-quarter" id="keyInfo">
+        {{ keyInfoItems(keyInfo) }}
+      </div>
```

Comment:
See comment above for the component. The `<div` tag should be in the component because otherwise the CSS rules are dependent on both template *and* component. I think they really ought to apply to the component in isolation.

Reply:
again, valid and I agree.

Reply:
updated.